src/main/resources/application.yml

```diff
@@ -70,6 +70,9 @@ delius:
   ldap:
     users:
       base: "cn=Users,dc=moj,dc=com"
+  baseurl: 'http://localhost:8088/alfresco/s/noms-spg'
+  username: 'A'
+  password: 'A'
```

Comment:
don't think we should have defaults for this in the application.yml

Reply:
changed my mind based on the `dgoss` stuff running.  When that gets removed then we can remove this